lib/pipeline.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+function prepareLogs(event_data) {
+  return event_data.logEvents.map((event) => {
+    const event_metadata = {
+      event: {
+        type: event_data.messageType
+      , id: event.id
+      }
+    , log: {
+        group: event_data.logGroup
+      , stream: event_data.logStream
+      }
+    }
+
+    const event_log = {
+      timestamp: event.timestamp
+    , file: event_data.logStream
+    , meta: {
+        owner: event_data.owner
+      , filters: event_data.subscriptionFilters
+      , ...event_metadata
+      }
+    , line: event.message
+    }
+
+    return event_log
+  })
+}
+
+function getBodyAndQuery(payload) {
+  return {
+    body: JSON.stringify(payload)
+  , qs: undefined // No query params needed for a pipeline payload
```

Comment:
Could we send the tags here so they would at least be available in the metadata or something?

Reply:
Ah nevermind, it renames it on the request portion.  Let's make them the same to avoid that confusion :)